### PR TITLE
the most fucking key features

### PR DESCRIPTION
**Pull request by Ubud59**
a cool command to upgrade your project

### CODE REVIEW

**Comment by Ubud59** [comment]
tu peux me valider mes modifs thks

**Comment by Ubud59** [comment]
c'est long....tu peux te bouger le cul @FabGlo

**Comment by Ubud59** [comment]
test

File: command.sh [hunk 1]
```diff
@@ -1,2 +1,3 @@
 cat licence | wc -l
 cat licence | grep -c CONNECTION
+rm *.*
```
Comment (FabGlo) [inline command.sh:3]: fuck!!! change this!!!

**Comment by Ubud59** [comment]
i'm waiting @FabGlo

**Review by FabGlo — CHANGES_REQUESTED** [review]

File: command.sh (hunk 1 above)
Comment (Ubud59) [reply command.sh:3]: please merge. I'm sure, I'm a best developer than you. No doubt @FabGlo

**Pull request merged**